lib/emerald/parser.rb

```diff
@@ -1,12 +1,20 @@
 module Emerald
   class Parser
+    INVALID_LIST_ERR = "Error - invalid list"
```

Comment:
Is the benefit of that in this instance to be able to categorise and keep track of the type of errors that can occur, and possibly handle them more appropriately?